resources/dicts/events/lifegen_events/murder.json

```diff
@@ -160,8 +160,8 @@
         "You tell v_c about a new herb to ease their aches and ask them to try it. It's actually water hemlock. Within hours, v_c succumbs to the poison."
     ],
     "murder mediatorapprentice kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
-        "As you come to realize how easily your voice can control others, you test this out on a simple minded kit - v_c to be precise! Lately, the young kit have been going through some zoomies, and with their rowdy energy, you propose the tiny kit to spend their energy in a burrow hole not too far from the camp, adding that they can play there as much as they want without any supervision. With a bright smile and a pep in their step as they thank you, you wave them goodbye, knowing, even though v_c has a spunky energy, their tiny legs wouldn't save them from the hungry fox living in that burrow.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, already having joined StarClan.",
+        "As you come to realize how easily your voice can control others, you test this out on a simple minded kit - v_c to be precise! Lately, the young kit have been going through some zoomies, and with their rowdy energy, you propose the tiny kit to spend their energy in a burrow hole not too far from the camp, adding that they can play there as much as they want without any supervision. With a bright smile and a pep in their step as they thank you, you wave them goodbye, knowing v_c wouldn't be making it back.",
```

Comment:
"you test this out on a simple minded kit - v_c to be precise!"
to
"you test this out on a simple-minded kit - v_c, to be precise!"

"Lately, the young kit have been going through some zoomies, and with their rowdy energy, you propose the tiny kit to spend their energy in a burrow hole not too far from the camp, adding that they can play there as much as they want without any supervision."
to
"Lately, the young kit has had a case of the zoomies, and you propose they expend their energy in a burrow not far from camp. You sweeten the deal by telling them they can play as much as they want there with no supervision!"

"With a bright smile and a pep in their step as they thank you, you wave them goodbye, knowing v_c wouldn't be making it back."
to
"With a bright smile and a pep in their step, v_c thanks you and heads into the burrow. You say your goodbyes, turning away with the knowledge that the kit wouldn't be coming back out."